src/app/pages/products-list/card/product-card.component.ts

```diff
@@ -0,0 +1,22 @@
+import {Component, OnInit} from '@angular/core';
+import {IProduct} from 'src/app/shared/products/product.interface';
+import {productMock} from 'src/app/shared/products/product.mock';
+
+@Component({
+    selector: 'app-product-card',
+    templateUrl: './product-card.component.html',
+    styleUrls: ['./product-card.component.css'],
+})
+export class ProductCardComponent implements OnInit {
+    product: IProduct | undefined;
+
+    ngOnInit(): void {
+        this.product = productMock;
+    }
```

Comment:
Почему так? Нужен ли нам тут хук, если мы в конструкторе можем произвести инициализацию?
```suggestion
    readonly product = productMock;
```

Reply:
Не нужен, убрал:)